lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -60,60 +63,65 @@ Function Chocolatey-Install-Upgrade
     $ChocoAlreadyInstalled = get-command choco -ErrorAction 0
     if ($ChocoAlreadyInstalled -eq $null)
     {
+
         #We need to install chocolatey
         $install_output = (new-object net.webclient).DownloadString("https://chocolatey.org/install.ps1") | powershell -
         if ($LASTEXITCODE -ne 0)
         {
-            Set-Attr $result "choco_bootstrap_output" $install_output
+            $result.choco_bootstrap_output = $install_output
             Fail-Json $result "Chocolatey bootstrap installation failed."
         }
         $result.changed = $true
         $script:executable = "C:\ProgramData\chocolatey\bin\choco.exe"
+
     }
     else
     {
+
         $script:executable = "choco.exe"
 
-        if ([Version](choco --version) -lt [Version]'0.9.9')
+        if ([Version](choco --version) -lt [Version]'0.9.9')xi
```

Comment:
Damn, these are VIM commands :-)
Part of cleaning up the formatting. What's weird is that CI didn't trip on this ?
Fixed.